webpack/assets/javascripts/react_app/common/Tour/BasicWrappedComponent.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { noop } from '../helpers';
+
+class BasicWrappedComponent extends Component {
```

Comment:
I'm not sure I got it. Why do we need the `BasicWrappedComponent` and the `BasicTour`?